### ISSUE TITLE
Project needs readme

### ISSUE THREAD

**Issue by mongrelion** [Rubberkat/Project-NS-vertrektijden#1]
Right now it is very difficult to tell what the project is about. A README file is not only helpful for future contributors to collaborate with the project but also is a good entry point for new users that want to use the product

**Comment by jonkoops** [comment]
I agree, it is very unclear what I need to do. I heard this was the best NS library, but I don't know. Hahahah.

**Comment by samuelleeuwenburg** [comment]
@jonkoops there is a fork up somewhere, they are going to merge back into this one somewhere around `0.1.x` however I can't find it at the moment

**Comment by jonkoops** [comment]
@samuelleeuwenburg That is nice. But my project deadline is very important!! Can you fix it now?? :8ball:

**Comment by mongrelion** [comment]
Who has direct contact with @Rubberkat? perhaps we can set a deadline to get the ball rolling?

**Comment by jonkoops** [comment]
I suggest we make a milestone. :+1:

**Comment by mongrelion** [comment]
:+1:

**Comment by jonkoops** [comment]
@samuelleeuwenburg @Rubberkat Comments? :1234:

**Comment by samuelleeuwenburg** [comment]
@jonkoops LGTM
